agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/statsbeat/Instrumentations.java

```diff
@@ -79,20 +79,29 @@ class Instrumentations {
     INSTRUMENTATION_MAP.put("io.opentelemetry.netty-4.0", 43);
     INSTRUMENTATION_MAP.put("io.opentelemetry.netty-4.1", 44);
     INSTRUMENTATION_MAP.put("io.opentelemetry.okhttp-3.0", 45);
-    INSTRUMENTATION_MAP.put("io.opentelemetry.opentelemetry-annotations-1.0", 46);
+    INSTRUMENTATION_MAP.put("io.opentelemetry.opentelemetry-extension-annotations-1.0", 46);
     INSTRUMENTATION_MAP.put("io.opentelemetry.akka-http-10.0", 47);
     INSTRUMENTATION_MAP.put("io.opentelemetry.spring-webmvc-3.1", 48);
     INSTRUMENTATION_MAP.put("io.opentelemetry.spring-webflux-5.0", 49);
     INSTRUMENTATION_MAP.put("io.opentelemetry.reactor-netty-1.0", 50);
     INSTRUMENTATION_MAP.put("io.opentelemetry.servlet-2.2", 51);
     INSTRUMENTATION_MAP.put("io.opentelemetry.servlet-3.0", 52);
-    INSTRUMENTATION_MAP.put("io.opentelemetry.servlet-common", 53);
     INSTRUMENTATION_MAP.put("io.opentelemetry.spring-scheduling-3.1", 54);
     INSTRUMENTATION_MAP.put("io.opentelemetry.play-2.4", 55);
     INSTRUMENTATION_MAP.put("io.opentelemetry.play-2.6", 56);
     INSTRUMENTATION_MAP.put("io.opentelemetry.vertx-http-client-3.0", 57);
     INSTRUMENTATION_MAP.put("io.opentelemetry.vertx-http-client-4.0", 58);
     INSTRUMENTATION_MAP.put("io.opentelemetry.spring-jms-2.0", 59);
+    INSTRUMENTATION_MAP.put("io.opentelemetry.tomcat-10.0", 60);
+    INSTRUMENTATION_MAP.put("io.opentelemetry.jetty-11.0", 61);
+    // TODO (trask) the liberty names should be getting a version soon
+    // see https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/6449
+    INSTRUMENTATION_MAP.put("io.opentelemetry.liberty", 62);
+    INSTRUMENTATION_MAP.put("io.opentelemetry.liberty-dispatcher", 63);
+    INSTRUMENTATION_MAP.put("io.opentelemetry.methods", 64); // used by "custom instrumentation"
+    INSTRUMENTATION_MAP.put("io.opentelemetry.okhttp-2.2", 65);
```

Comment:
this is exceeding 64 bits. do we want to send second long for anything that is > 64? or clean some up?

Reply:
ya, that's a good idea to prolong adding the second long

Reply:
I went through all the indexes (and found more missing instrumentations), and if we reuse the indexes that we retired in 3.3.0, we will be completely maxing out the 64 bits, and one more instrumentation will bump us to a second long.

Since the second long seems inevitable anyways (and probably soon), I think we may as well do that now and avoid reusing which means more version-specific mappings.

Reply:
i will have a follow-up PR to send second long.